app/views/public_recipe/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="m-20">
+  <h1 class="mb-11 text-3xl">Public Recipes</h1>
+
+  <div id="recipes">
+<% @foods.each do |food| %>
+    <% food.user.sortrecipes.each_with_index do |recipe| %>
+        <% if recipe.public%>
+        <div class="flex items-center justify-between mt-5 border-solid w-3/4 rounded-md border-2 borde py-2 px-5 bg-gray-50 hover:bg-slate-400 text-center">
+                <div class="bg-white border-b-2 text-slate-700 border-slate-700">
+                    <h4 class="p-4 text-xl font-medium">the recipe is: <%= recipe.name %></h1>
+                    <h3 class="p-4 text-xl">By: <%= food.user.name%></h3>
+                </div>
+                <div class="flex flex-col">
+                    <h2 class="my-5 border-solid w-72 rounded-md p-2 bg-black text-white text-xl text-center">amought of items to buy: <%= @foods.length%></h2>
+                    <h3 class="my-5 border-solid w-80 rounded-md p-2 bg-black text-white text-xl text-center">total value of food needed it:
+                    <%=  food.quantity %></h3>
+                </div>
+                    <p class="my-7 border-solid w-56 rounded-md border-2 borde  r-slate-200 p-4 bg-gray-50 hover:bg-black hover:text-white text-center">
+                        <%= link_to "recipe details", user_recipe_path(food.user.id,recipe.id) %>
+                    </p>
+        </div>
+        <%end%>
+    <%end%>
```

Comment:
- [ ] As required by the previous code reviewer please 🙏🏽 match with the wireframe, kindly align the content at the center of the page.